src/templatizer/dom-repeat.html

```diff
@@ -397,31 +406,29 @@
         // so queue a full refresh
         this._needFullRefresh = true;
         this._initializeChunking();
-        this._debounceRender();
+        this._debounceRender(this._render);
       }
-    },
+    }
 
-    _checkObservedPaths: function(path) {
+    _checkObservedPaths(path) {
       if (this._observePaths) {
         path = path.substring(path.indexOf('.') + 1);
         var paths = this._observePaths;
         for (var i=0; i<paths.length; i++) {
           if (path.indexOf(paths[i]) === 0) {
             this._needFullRefresh = true;
-            if (this.delay) {
-              this.debounce('render', this._render, this.delay);
-            } else {
-              this._debounceRender();
-            }
+            this._debounceRender(this._render, this.delay);
             return true;
           }
         }
       }
-    },
+    }
 
-    _debounceRender: function() {
-      Polymer.Templatizer.enqueueDebouncer(this.debounce('_render', this._render));
-    },
+    _debounceRender(fn, delay) {
```

Comment:
In `dom-if` a similar function is called `queueRender`. Any reason to name this differently?